main.py

```diff
@@ -1,17 +1,18 @@
+import pandas as pa
+
 from db_connection import build_full_url, make_engine, get_session
 from db_engines.sql_server_engine import url_SQLServerTestDBMS
 from db_engines.db_sources_data.sql_server_test_localhost import dbdriver, dbpath, username, password, SQLServerTestDBs
 
 from loguru import logger
 from loguru_logging.debug_formatter import debug_format
 
+import dql_scripts
 import dql_scripts.simple_select as smpl_sel
 import dql_scripts.select_joins as jnt_sel
-from ddl_scripts.creating_tables import SignalMeta
-from ddl_scripts.adding_data import add_new_objs, test_dataobj1, test_dataobj2
 
 debug_format()
-url = build_full_url(url_SQLServerTestDBMS + SQLServerTestDBs.MASTER_DB.value + "?driver=ODBC+Driver+17+for+SQL+Server")
+url = build_full_url(url=url_SQLServerTestDBMS + SQLServerTestDBs.MASTER_DB.value, odbc_driver="ODBC Driver 17 for SQL Server")
```

Comment:
Here we have build_full_url and make_engine but but in sql_server_engine we created another engine called: engine_SQLServerTest_MainDB. Does it make sense to create one session in main ?